cbw_api_toolbox/cbw_api.py

```diff
@@ -60,7 +61,7 @@ def __init__(
         self.logger = logging.getLogger(self.__class__.__name__)
 
     def _build_route(self, params):
-        return "{0}{1}".format(self.api_url, '/'.join(params))
+        return join(self.api_url, *params)
```

Comment:
Oula, il se passe quoi sur python sur windows ?

Reply:
You are right, it doesn't work at all
```python
>>> os.path.join("https://localhost", "api/v2")
'https://localhost\\api/v2'
```

Reply:
using `urllib.parse.urljoin`:
```python
>>> from urllib.parse import urljoin
>>> urljoin("https://localhost//", "/api/v2")
'https://localhost/api/v2'
```